packages/backend/src/payment-method/ilp/connector/core/middleware/reduce-expiry.ts

```diff
@@ -24,7 +24,7 @@ function getDestinationExpiry(
   const sourceExpiryTime = request.expiresAt.getTime()
 
   if (sourceExpiryTime < Date.now()) {
-    log.trace('incoming packet has already expired', { request })
+    log.debug({ request }, 'incoming packet has already expired')
```

Comment:
pino doesnt complain, but `{request}` did not end up getting logged when put in the original order.